### ISSUE TITLE
Feature: Add security.txt

### ISSUE THREAD

**Issue by ingben** [sinnerschrader/website#306]
## About

Security.txt is an IETF draft that aims to define a standard for security researchers to report security findings responsibly and in a consistent manner. For details, please check https://www.ietf.org/id/draft-foudil-securitytxt-00.txt.

In order to implement this feature, we just have to place a file named 'security.txt' in the root folder of the website.

## Actual behaviour

Accessing https://sinnerschrader.com/security.txt returns an HTTP 404.

## Expected behaviour

Accessing https://sinnerschrader.com/security.txt returns a text file (yes, no HTML, JS, CSS) with the following content.

```
# Hey there! Please follow responsible disclosure.

Contact: security@sinnerschrader.com
Encryption: https://pgp.mit.edu/pks/lookup?op=get&search=0x51E4E22B107D719A
Disclosure: Full
```

**Comment by marionebl** [comment]
Thanks for the suggestion, @ingben.

The contents of `/static` in repository root are copied to document root during build, so placing the contents as described at `/static/.well-known/security.txt` should do the trick.

Care to lend a hand?

**Comment by ingben** [comment]
Sure. Like so? #307 ?

**Comment by marionebl** [comment]
Yep, thanks! I'll have a look and smoothen out build problems if any.
